packages/ruleset/test/description-mentions-json.test.js

```diff
@@ -50,7 +50,7 @@ describe('Spectral rule: description-mentions-json', () => {
         'This is a JSON object.';
 
       const results = await testRule(ruleId, rule, testDocument);
-      expect(results).toHaveLength(2);
+      expect(results).toHaveLength(3);
```

Comment:
now there's an additional occurrence of this run due to the new operation in the root document